### PR TITLE
Rename and update to torch 1.11

### PR DESCRIPTION
**Pull request by loadams**
torch 1.10 is fairly out of date at this point, this moves the oldest versions we test up to 1.13.1.

P40 tests: https://github.com/microsoft/DeepSpeed/actions/runs/7934885228/job/21666817397?pr=5141
V100 tests: https://github.com/microsoft/DeepSpeed/actions/runs/7934885226/job/21666817704?pr=5141

### CODE REVIEW

**Review by mrwyattii — APPROVED** [review]

**Review by mrwyattii — APPROVED** [review]

File: .github/workflows/nv-torch113-p40.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
-name: nv-torch110-p40
+name: nv-torch113-p40
 
 on:
   workflow_dispatch:
+  pull_request:
```
Comment (loadams) [inline .github/workflows/nv-torch113-p40.yml:5]: Remove this before completing PR.

File: .github/workflows/nv-torch113-v100.yml [hunk 2]
```diff
@@ -1,7 +1,8 @@
-name: nv-torch110-v100
+name: nv-torch113-v100
 
 on:
   workflow_dispatch:
+  pull_request:
```
Comment (loadams) [inline .github/workflows/nv-torch113-v100.yml:5]: Remove this before completing PR.

File: .github/workflows/nv-torch113-p40.yml [hunk 3]
```diff
@@ -1,7 +1,8 @@
-name: nv-torch110-p40
```
Comment (loadams) [inline .github/workflows/nv-torch113-p40.yml:1]: Update this to test 1.11 as this is the latest supported by transformers/accelerate.

Comment (loadams) [reply .github/workflows/nv-torch113-p40.yml]: Done.

**Pull request closed without merging**